core/src/main/java/uk/ac/bbsrc/tgac/miso/core/manager/MisoRequestManager.java

```diff
@@ -1221,7 +1233,9 @@ public long saveRun(Run run) throws IOException {
   @Override
   public long saveRunQC(RunQC runQC) throws IOException {
     if (runQcStore != null) {
-      return runQcStore.save(runQC);
+      long id = runQcStore.save(runQC);
+      if (runAlertManager != null) runAlertManager.update(runQC.getRun());
```

Comment:
The reason I have this checks is so that we don't have to wire them for migration.